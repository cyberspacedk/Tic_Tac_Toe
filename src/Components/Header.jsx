import React from 'react';
import styled from "styled-components";

const HeaderTop = styled.div`
background-color:#3c3c3c;
height:100px;
width:100vw;
line-height:100px;
text-align:center;
color: #fff;
font-size: 2rem;
box-shadow: 1px 4px 13px 0px #676767;

`;


const Header = () => {
    return (
        <HeaderTop>
            TIC TAC TOE
        </HeaderTop>
    );
}

export default Header;
