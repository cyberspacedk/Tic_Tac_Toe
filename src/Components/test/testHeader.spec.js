import React from "react";
import { shallow } from "enzyme";
import { shallowToJson } from "enzyme-to-json"; 
import Header from "../Header";

describe("Header component", () => {

  // 1- check. content should be - TIC TAC TOE
  // 2- snapshot

  const myComponent = shallow(<Header />);

  it("check. content should be - TIC TAC TOE", () => {
    expect(myComponent.text()).toEqual("TIC TAC TOE");
  });

  it('Header snapshot', ()=>{
    expect(shallowToJson(myComponent)).toMatchSnapshot()
  })
 
});
