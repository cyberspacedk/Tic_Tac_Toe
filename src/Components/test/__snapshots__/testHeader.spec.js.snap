// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component Header snapshot 1`] = `
<styled.div>
  TIC TAC TOE
</styled.div>
`;
